src/components/molecules/Visualizer/Infobox/Frame/index.tsx

```diff
@@ -98,13 +101,23 @@ const InfoBox: React.FC<Props> = ({
           justify={open ? "flex-start" : "space-evenly"}
           direction="column"
           onClick={handleOpen}>
-          {isSmallWindow && !noContent && <StyledIcon icon="arrowUp" size={24} open={open} />}
+          {isSmallWindow && !noContent && (
+            <StyledIcon publishedTheme={publishedTheme} icon="arrowUp" size={24} open={open} />
+          )}
           <Text size="m" weight="bold" customColor>
-            <TitleText>{title || " "}</TitleText>
+            <TitleText publishedTheme={publishedTheme}>{title || " "}</TitleText>
```

Comment:
Is this needed? Since the wrapper is getting the publishedTheme styles and the Text component that is wrapping this span is set to `customColor`, I feel like this shouldn't be needed.
If it is needed because the span isn't allowing the styles to be used let me know.

Reply:
Totally right, it's a mistake I made after resolving the conflicts.
Will fix.